src/Classes/ModDB.lua

```diff
@@ -90,14 +90,23 @@ end
 
 function ModDBClass:SumInternal(context, modType, cfg, flags, keywordFlags, source, ...)
 	local result = 0
+	local globalLimits = { }
```

Comment:
I can't think of any examples of this so it's not necessary to fix, but if there is ever a mod that has a parent and a global limit, they won't share the same `globalLimits` table as far as I can tell.